common/common.cpp

```diff
@@ -61,7 +64,7 @@
 #else
 #include <sys/syslimits.h>
 #endif
-#define LLAMA_CURL_MAX_PATH_LENGTH PATH_MAX
+#define LLAMA_CURL_MAX_URL_LENGTH 2084 // Maximum URL Length in Chrome: 2083
 #define LLAMA_CURL_MAX_HEADER_LENGTH 256
```

Comment:
I do not use `CURL_MAX_HTTP_HEADER` because it is `100*1024` which is IMHO too big for `ETag: xxxx' and `Last-Modified: date`